helm/examples/azure/hpcc-azurefile/templates/NOTES.txt

```diff
@@ -1,8 +1,9 @@
 {{ $dllPvcName := printf "dll-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
 {{ $daliPvcName := printf "dali-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
 {{ $dataPvcName := printf "data-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
+{{ $sashaPvcName := printf "sasha-%s-pvc" (include "hpcc-azurefile.fullname" $) }}
```

Comment:
should be removed, see comment below.

Reply:
I will follow the pattern in localfile NOTES.txt